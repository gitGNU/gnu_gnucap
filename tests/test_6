*requirement:sysinfo shared object file should be loaded. 
echo ______________test sysinfo command______________
sysinfo
