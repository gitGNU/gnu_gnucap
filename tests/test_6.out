______________test sysinfo command______________ 
Model name	: Intel(R) Core(TM) i5-2450M CPU @ 2.50GHz
OS Description  :"Ubuntu 12.04 LTS"
CPU MHz		: 2494.389
RAM Available   : 1542776kB
RAM Free        : 725376kB
